lib/00_hello.rb

```diff
@@ -1,11 +1,10 @@
 def ask_first_name
-  puts "What's you beautiful name?"
-  print "❯ "
-  return String(gets.chomp)
+  print "What's you beautiful name?\n > "
+  return (gets.chomp)  
 end
 
 def say_hello(first_name)
   puts "So hello, #{first_name}"
 end
 
-say_hello(ask_first_name)
\ No newline at end of file
+say_hello(ask_first_name)
```

Comment:
```suggestion
first_name = ask_first_name()
say_hello(first_name)
```

C'est plus lisible comme ça. Quand tu commencera à créer des programmes plus complexes, c'est ce genre de pattern que tu utilisera.

Reply:
Ça me permet de lever le doute sur l'utilisation des fonctions. Je pensais qu'on devait les appeler de la façon suivante `foo` et qu'on ne pouvait pas de cette façon là `foo()` *comme en javascript par exemple*.
Ça veut aussi dire qu'on peut définir une fonction de la façon suivante ?
```rb
def ma_fonction()
  # mon code
end
```

Reply:
Oui tu peux. De préférence, quand tu appels ta fonction, alors il vaut mieux mettre des parenthèses.

Quand tu définies une fonction, alors tu ne met des parenthèses que si cette même fonction prend un ou plusieurs arguments.

Avec Ruby, les parenthèse ne sont pas obligatoires (cf `print` et `puts` dans ton code), mais il y a quand même quelques règles que la communauté préfère suivre pour des raisons de lisibilité. Si ça t'intéresse, tu peux te référer aux différent _Ruby Style Guide_:

- https://github.com/rubocop-hq/ruby-style-guide
- https://shopify.github.io/ruby-style-guide/
- https://github.com/airbnb/ruby